src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -337,24 +334,24 @@
   "entity.minecolonies.archerbarbarian": "Archer Barbarian",
   "entity.minecolonies.chiefbarbarian": "Chief Barbarian",
   "entity.chiefbarbarian": "Chief Barbarian",
-  "death.attack.entity.minecolonies.barbarian": "%s was Sliced & Diced by a Barbarian!",
-  "death.attack.entity.minecolonies.chiefbarbarian": "%s was Macerated by a Great Chief!",
-  "death.attack.entity.minecolonies.archerbarbarian": "%s was ravaged by an Archer Barbarian",
-  "entity.archerbarbarian": "Barbarian Archer",
+  "death.attack.entity.minecolonies.barbarian": "%s was sliced and diced by a Barbarian!",
+  "death.attack.entity.minecolonies.chiefbarbarian": "%s was macerated by a Great Chief!",
+  "death.attack.entity.minecolonies.archerbarbarian": "%s was ravaged by an Archer Barbarian!",
+  "entity.archerbarbarian": "Archer Barbarian",
 
-  "entity.builder.messagenobuilder": "You need a builder in order to build or upgrade this building.",
+  "entity.builder.messagenobuilder": "You need a Builder to build or upgrade this building.",
   "entity.farmer.needseed": "I need %s to plant.",
 
-  "com.minecolonies.command.help.wiki": "Checkout the wiki to learn more: ",
+  "com.minecolonies.command.help.wiki": "Check out the wiki to learn more: ",
   "com.minecolonies.command.help.discord": "Join the community at  ",
-  "com.minecolonies.command.notop": "Must be OP to use the command",
-  "com.minecolonies.command.whereami.nocolony": "No colony close at all.",
-  "com.minecolonies.command.whereami.colonyclose": "You're not inside any colony, the closest colony is approx %.2f blocks away.",
-  "com.minecolonies.command.whereami.incolony": "You're inside colony %s with id: %s, the colony center is approx %.2f blocks away.",
-  "com.minecolonies.command.whoami.hascolony": "You are %s, your colony is %s, with ID:%d at position %s",
-  "com.minecolonies.command.whoami.nocolony": "You are %s, you do not have a colony yet.",
-  "com.minecolonies.command.rtp.notallowed": "You're not allowed to teleport, change serverconfig to enable it.",
-  "com.minecolonies.command.rtp.wrongdim": "random teleport is only allowed on the overworld.",
+  "com.minecolonies.command.notop": "Must be OP to use this command",
+  "com.minecolonies.command.whereami.nocolony": "No close colony.",
+  "com.minecolonies.command.whereami.colonyclose": "You're not inside any colony. The closest colony is approx %.2f blocks away.",
+  "com.minecolonies.command.whereami.incolony": "You're inside colony %s with id: %s. The colony center is approx %.2f blocks away.",
```

Comment:
Having used this command before, I am suspicious. I don't believe this is actually measured in blocks.

Reply:
I think it is:

![2020-08-06_11 57 23](https://user-images.githubusercontent.com/63612548/89560001-01ba8780-d7dc-11ea-931f-ec5e35c2d72e.png)

Reply:
Alright, well either I was imagining it, or it's glitchy. Either way the translation for it is fine then.